syncopy/specest/_norm_spec.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Helper routines to normalize Fourier spectra
+#
+
+import numpy as np
+
+
+def _norm_spec(ftr, nSamples, freqs):
+
+    """
+    Normalizes the complex Fourier transform to
+    power spectral density units.
+    """
+
+    # frequency bins
+    delta_f = freqs[1] - freqs[0]
+    ftr = ftr / (nSamples / 2 * np.sqrt(delta_f))
```

Comment:
Ah, now I finally understand this.. it's not merely syntactic sugar, but we save one (temporary) array which otherwise would have been created on the rhs :)

Reply:
That answer is a little misleading in my opinion. This is true for `iadd` and `add` defined on regular types or classes, not necessarily collections. NumPy arrays are a bit of a different beast due to the underlying CPython pointer magic. See [here](https://stackoverflow.com/questions/35910577/why-does-python-numpys-mutate-the-original-array) for a more recent SO thread.